cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -2535,7 +2556,8 @@ search = `cs_o365` sourcetype="o365:management:activity" _index_earliest=-61m@m
 | `cs_current_week_login_count_o365` \
 | `cs_country_login_percentage` \
 | fillnull Country, Region, City value="-" \
-| eval cyences_severity = case(percentage_login_from_country==0, "high", user_type=="Admin" or user_type=="DcAdmin", "high", true(), "medium") \
+| `cs_user_privilege_mapping(user)` \
+| eval cyences_severity = case(percentage_login_from_country==0, "high", user_type=="Admin" or user_type=="DcAdmin" or is_privileged_user=="Yes", "high", true(), "medium") \
```

Comment:
critical

Reply:
In which case, you are asking to make it critical? **percentage_login_from_country==0** or **user_type=="Admin" or user_type=="DcAdmin" or is_privileged_user=="Yes"**

FYI, previously, we decreased the severity from critical to high for **percentage_login_from_country==0** (if new user login attempt)

Reply:
Lets do this:
* percentage_login_from_country==0 and (user_type=="Admin" or user_type=="DcAdmin" or is_privileged_user=="Yes") -> critical
* percentage_login_from_country>0 and (user_type=="Admin" or user_type=="DcAdmin" or is_privileged_user=="Yes") -> high
* percentage_login_from_country==0 and NOT (user_type=="Admin" or user_type=="DcAdmin" or is_privileged_user=="Yes") -> high
* as it is otherwise